patches/api/0407-Expand-Instrument-API.patch

```diff
@@ -0,0 +1,62 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jake Potrebic <jake.m.potrebic@gmail.com>
+Date: Sat, 19 Nov 2022 12:38:15 -0800
+Subject: [PATCH] Expand Instrument API
+
+
+diff --git a/src/main/java/org/bukkit/Instrument.java b/src/main/java/org/bukkit/Instrument.java
+index 92194803bcdfbcfdb99567835906ce9219eabd04..6fbdc2a177e7361e2e3e55d47714deec5944f907 100644
+--- a/src/main/java/org/bukkit/Instrument.java
++++ b/src/main/java/org/bukkit/Instrument.java
+@@ -78,6 +78,7 @@ public enum Instrument {
+ 
+     private final byte type;
+     private static final Map<Byte, Instrument> BY_DATA = Maps.newHashMap();
++    private static final java.util.Optional<org.bukkit.Instrument.InstrumentUtil> UTIL = net.kyori.adventure.util.Services.service(InstrumentUtil.class); // Paper
```

Comment:
if we are only throwing on call anyways, might as well throw here, once, right? if we cant find that class, might as well stop start startup since something is fucked

Reply:
Api tests don’t work then

Reply:
fiar enough